config/routes.rb

```diff
@@ -24,12 +24,14 @@
   devise_scope :user do
     post "/users/invitation/resend/:id" => "invitations#resend", :as => "resend_user_invitation"
     put "/users/confirmation" => "confirmations#update"
+    resource :two_step_verification_session,
```

Comment:
I used `rails routes | grep two_step_verification` before and after this commit to confirm that there are no changes to the Rails-generated URL helper methods :+1: